content/millennium/r4/individuals/related-person.md

```diff
@@ -0,0 +1,105 @@
+
+The distinction between Practitioner and RelatedPerson is a practitioner cares for multiple patients on behalf of a healthcare facility where a RelatedPerson performs care tasks for a specific patient and is not associated with any healthcare facility.
+
+* [RelatedPerson id](http://hl7.org/fhir/R4/resource-definitions.html#Resource.id){:target="_blank"}
+* [RelatedPerson Identifier](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.identifier){:target="_blank"}
+* [Active](https://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.active){:target="_blank"} 
+* [Patient](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.patient){:target="_blank"}
+* [Relationship](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.relationship){:target="_blank"}
+* [Related person's name](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.name){:target="_blank"}
+* [Telecom information (may include phone and email)](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.telecom){:target="_blank"}
+* [Gender (administrative)](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.gender){:target="_blank"}
+* [Date of birth](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.birthDate){:target="_blank"}
+* [Address](http://hl7.org/fhir/R4/relatedperson-definitions.html#RelatedPerson.address){:target="_blank"}
+* [Communication (preferred language)](https://hl7.org/fhir/r4/relatedperson-definitions.html#RelatedPerson.communication){:target="_blank"}
+
+## Terminology Bindings
+
+<%= terminology_table(:related_person, :r4) %>
+
+## Search
+
+Search for RelatedPersons that meet supplied query parameters:
+
+    GET /RelatedPerson?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name         | Required?                                         | Type          | Description
+--------------|---------------------------------------------------|-----------------------------------------------------------------------------------------------------------
+ `_id`        | This or `patient` or `identifier` | [`token`]     | The logical resource id associated with the resource.
+ `identifier` | This or `_id` or `patient`        | [`token`]     | A RelatedPerson identifier.
+ `patient`    | This or `_id` or `identifier`     | [`reference`] | A reference to a patient related which the RelatedPerson is associated. Example: `14067892`
+
+Notes:
+
+- `identifier` value must include both a system and a code. Example: `identifier=urn:oid:2.16.840.1.113883.3.13.6|URN:CERNER:...:PI98N2FK5TN`
```

Comment:
"When provided, the `identifier` query parameter must include..."

use a full example instead of an ellipsis in the code

Reply:
https://github.com/cerner/fhir.cerner.com/pull/254/commits/53300d0b6211f4b0c77925c1e5ab9b78e4010aff#diff-b47a4cc256449f5aa96d433a01e1ae01R54

Reply:
"- When provided, the `identifier` query parameter must include both a system and a code. Example: `identifier=urn:oid:2.16.840.1.113883.3.13.6|RANDES011E8442-2E86-4A00-8394-EFC6A3434A8F`"

Reply:
Updated
https://github.com/cerner/fhir.cerner.com/pull/254/commits/f89cf3b7e8dada9855a83b559e742ca198699db6#diff-b47a4cc256449f5aa96d433a01e1ae01R54